scripts/upload_s2_nbart.py

```diff
@@ -110,7 +111,7 @@ def upload_granule(granule_id):
 
         publish_sns(SNS_ARN, stac_dump, message_attributes, session=session)
 
-        s3_dump(stac_dump, s3_stac_path)  # upload STAC last
+        s3_dump(stac_dump, s3_stac_path, ACL="bucket-owner-full-control")  # upload STAC last
```

Comment:
Can you please set ContentType for this.
```suggestion
        s3_dump(stac_dump, s3_stac_path, ACL="bucket-owner-full-control", ContentType="application/json")  # upload STAC last
```